client/ayon_fusion/plugins/publish/collect_render.py

```diff
@@ -37,10 +37,8 @@ def get_instances(self, context):
         aspect_x = comp_frame_format_prefs["AspectX"]
         aspect_y = comp_frame_format_prefs["AspectY"]
 
-
         current_file = context.data["currentFile"]
-        version = context.data["version"]
-
+        version = context.data.get("version")
```

Comment:
Is this only really changed line?

Reply:
Yes correct - this is the only relevant change. Sorry for the mess with other 'cosmetic' changes.